packages/babel-plugin-transform-block-scoping/test/fixtures/general/for-break-continue-return/output.js

```diff
@@ -15,18 +15,11 @@
     }
   };
 
-  _loop: for (var i in nums) {
-    var _ret = _loop2(i);
+  for (var i in nums) {
+    var _ret = _loop(i);
 
-    switch (_ret) {
-      case "continue":
-        continue;
-
-      case "break":
-        break _loop;
-
-      default:
-        if (typeof _ret === "object") return _ret.v;
-    }
+    if (_ret === "continue") continue;
```

Comment:
Not related to this PR: the string literal here is not minimizer-friendly. We should revise https://github.com/babel/babel/blob/b1a8e72e169b967968c403242a7073b8122d0288/packages/babel-plugin-transform-block-scoping/src/index.js#L266-L272

later, e.g. change it to `"b"`/`"c"`.